lib/msal-browser/src/event/EventHandler.ts

```diff
@@ -13,11 +13,15 @@ export class EventHandler {
     private eventCallbacks: Map<string, EventCallbackFunction>;
     private logger: Logger;
     private browserCrypto: ICrypto;
+    private listeningToStorageEvents: boolean;
+    private boundStorageListener: (e: StorageEvent) => void;
 
     constructor(logger: Logger, browserCrypto: ICrypto) {
         this.eventCallbacks = new Map();
         this.logger = logger;
         this.browserCrypto = browserCrypto;
+        this.listeningToStorageEvents = false;
+        this.boundStorageListener = this.handleAccountCacheChange.bind(this);
```

Comment:
Can the name for these be closer? It is acceptable/idiomatic in JS to do this:

```suggestion
        this.handleAccountCacheChange = this.handleAccountCacheChange.bind(this);
```

Reply:
Done